src/Datastore/Blob.php

```diff
@@ -17,6 +17,7 @@
 
 namespace Google\Cloud\Datastore;
 
+use Google\Cloud\Core;
```

Comment:
No, but is there a reason to do it that way? `CoreBlob` and `Core\Blob` seem pretty interchangable.

Reply:
Because there is precedent elsewhere for this approach: https://github.com/GoogleCloudPlatform/google-cloud-php/pull/693/files#diff-59fb291be43a55572f01709321556b86R45